### DISCUSSION TITLE
Total Price from quantity on product page

### DISCUSSION THREAD

**Discussion by Danieleeffe1** [OpenMage/magento-lts#2175] [Ideas]
Goodmorning everyone.

I recently had the opportunity to deal with various E-commerce.

Some of these offer the possibility on the product page to obtain a quick quote on the total price of the product displayed based on the quantity entered by the customer before placing the product in the cart.

Magento / OpemnMage on the product page displays the unit price, example:
TSHIRT 10 €

I think it would be very useful (and current) to insert a function described above.

the prices to be taken and multiplied would obviously be

normal-price
special-price
tier-price
etc
etc
etc

What do you think?
Or is there already a module that does this?

**Comment by kiatng** [comment]
I needed exactly this feature, a total price on the product page. I solved it by adding a js in the phtml block on price rendering. In my case, I only need to take care of one specific product type with tier pricing. I didn't need to implement for all product types and for all possible price configurations. I think it's probably doable but not so straight forward.

**Comment by Danieleeffe1** [comment]
Hello

Thanks for the reply.

I am not a developer

kindly Could you share this your solution?

Thank you

**Comment by kiatng** [comment]
If you copy and paste the following, it has a very high chance of **not** working as you expect. That being said, insert the following script at the end of this file: `app\design\frontend\{ package name }\{ theme name }\template\catalog\product\view.phtml`

```html
<script>
optionsPrice.unitPrice = optionsPrice.productPrice;
if($('qty')) $('qty').observe('blur', function(e) {
    $('qty').value = Math.max($F('qty').replace(/[^\d].*/, ''), 1);
    optionsPrice.productPrice = Math.max(optionsPrice.unitPrice, $F('qty') * optionsPrice.unitPrice);
    optionsPrice.reload();
});
</script>
```

What it does:
First, it sets the unit price of the product to object `optionsPrice`. Then, when there is a change in qty, it recalculates the product price: `new price = qty * unitPrice`.

As I said, it will probably not work in your case without more customization.

**Comment by addison74** [comment]
@Danieleeffe1 - Did the code provide by @kiatng solved your issue? I would like to close since it has not had feedback for 1 year.

**Comment by Danieleeffe1** [comment]
I thank the community for trying to help me.

Unfortunately, the code provided did not have any satisfactory results
